### ISSUE TITLE
`cmake --build build/test --target format` not working

### ISSUE THREAD

**Issue by mscofield0** [TheLartians/ModernCppStarter#119]
I've installed all that the documentation said is required:
```
$ pip list
Package      Version
------------ -------
cmake-format 0.6.13
cmakelang    0.6.13
pip          21.0.1
PyYAML       5.4.1
setuptools   49.2.1
six          1.15.0
```
```
$ clang-format.exe --version
clang-format version 11.0.1
```

However, `cmake --build build/standalone --target format` works.

**Comment by TheLartians** [comment]
@mscofield0 the `format` target should error if your code is not properly formatted. `fix-format` should auto format it. Could you post the error message you are receiving to better understand the problem?

**Comment by mscofield0** [comment]
Yeah, here it is:

```
$ cmake --build build/test --target format
Microsoft (R) Build Engine version 16.9.0+5e4b48a27 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1009: Project file does not exist.
Switch: format.vcxproj
```

Also, I forgot to mention that I added the `Format.cmake` package to the standalone target.

**Comment by TheLartians** [comment]
That's strange, unfortunately I don't have much knowledge in MSVC, so it's hard to say why the project file doesn't exist. Does the error also occur if you create a fresh build of the `all` project? E.g. 

```bash
# delete build_all if it exists
cmake -S all -B build_all
cmake --build build_all --target format
```

May I ask why you added Format.cmake it to the standalone project?

**Comment by mscofield0** [comment]
> 
> 
> That's strange, unfortunately I don't have much knowledge in MSVC, so it's hard to say why the project file doesn't exist. Does the error also occur if you create a fresh build of the `all` project? E.g.
> 
> ```shell
> # delete build_all if it exists
> cmake -S all -B build_all
> cmake --build build_all --target format
> ```

Huh, that works. May I ask why it doesn't work on `build/test` though?

> May I ask why you added Format.cmake it to the standalone project?

No specific reason, I was new to the template so I wanted to play around with it.

**Comment by TheLartians** [comment]
> Huh, that works. May I ask why it doesn't work on build/test though?

Yeah, that is strange indeed. It should still work assuming that you didn't remove the Format.cmake target from the source there. What happens if you build the format target on the tests after a fresh clone of the template?

> No specific reason, I was new to the template so I wanted to play around with it.

Gotcha. Just FYI the idea of the Format target is to be run alongside the test suite and it will check the whole codebase committed to Git. As it isn't needed to build the standalone target, it makes most sense to be left alongside the tests.

**Comment by mscofield0** [comment]
> What happens if you build the format target on the tests after a fresh clone of the template?
These are the exact commands I entered:
```
$ git clone https://github.com/TheLartians/ModernCppStarter.git
$ cd ModernCppStarter
$ cmake -S test -B build/test
$ cmake --build build/test
$ cmake --build build/test --target format
Microsoft (R) Build Engine version 16.9.0+5e4b48a27 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1009: Project file does not exist.
Switch: test.vcxproj
```
Really can't tell what the issue might be...

> Gotcha. Just FYI the idea of the Format target is to be run alongside the test suite and it will check the whole codebase committed to Git. As it isn't needed to build the standalone target, it makes most sense to be left alongside the tests.
Oh, that makes sense. Thanks for the clarification.

**Comment by TheLartians** [comment]
Yeah, that's indeed very strange, the same command sequence works on my Mac without issues. I think this might be an issue with CMake, which version are you on? Unfortunately, I don't have access to a windows machine to explore this myself.

**Comment by mscofield0** [comment]
> I think this might be an issue with CMake, which version are you on?

I'm on CMake 3.20.0, which is the latest one.

**Comment by mscofield0** [comment]
This might be an important information, but the executable is in `build/test/Debug` instead of `build/test`

**Comment by AquilaSagitta** [comment]
On windows am having similar issues with test command too. Need to use command 
```
cmake --build build/test --target RUN_TESTS
```
Is there supposed to be a `FORMAT.vcxproj` file? This is what is in my build/test dir

![Image](https://github.com/user-attachments/assets/00bae10e-2f60-4a87-8c0f-aa990ba72590)
